# Request
string start_map
string target_map
---
# Response
string[] path
